orderer/common/cluster/clusterservice.go

```diff
@@ -130,30 +138,28 @@ func (s *ClusterService) VerifyAuthRequest(stream orderer.ClusterNodeService_Ste
 		Channel:        authReq.Channel,
 	})
 	if err != nil {
-		return nil, -1, errors.Wrap(err, "ASN encoding failed")
+		return nil, errors.Wrap(err, "ASN encoding failed")
 	}
 
-	s.Lock.RLock()
```

Comment:
no, please remove this :( 

We are RLocking when using VerifyRequest. Having two RLocks by the same goroutine will lead in a deadlock if there is a pending Lock in between from a different goroutine :)

Reply:
i removed it. I think, you are referring the old code, not the latest code.

Reply:
that's odd. when i clicked to only see that commit, it showed it still. When i clicked to see changes from all commits, it didn't show it.